setup/aio/011_Install_Puppet_Server.rb

```diff
@@ -0,0 +1,28 @@
+extend BeakerPuppet::Install::Puppet5
+extend Beaker::DSL::InstallUtils::FOSSUtils
+
+test_name "Install Puppet Server" do
+  server_version = ENV['SERVER_VERSION'] || 'latest'
+  release_stream = ENV['RELEASE_STREAM'] || 'puppet'
+  nightly_builds_url = ENV['NIGHTLY_BUILDS_URL'] || 'http://nightlies.puppet.com'
+  dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+
+  if nightly_builds_url == 'http://nightlies.puppet.com'
+    yum_nightlies_url = nightly_builds_url + '/yum'
+    apt_nightlies_url = nightly_builds_url + '/apt'
```

Comment:
This isn't quite working for apt

```
irb(main):001:0> require 'uri'
=> true
irb(main):003:0> opts = {}
=> {}
irb(main):004:0> opts[:release_apt_repo_url] = 'http://nightlies.puppet.com/apt'
=> "http://nightlies.puppet.com/apt"
irb(main):005:0> deb = 'puppet5-nightly-release-xenial.deb'
=> "puppet5-nightly-release-xenial.deb"
irb(main):006:0> deb
=> "puppet5-nightly-release-xenial.deb"
irb(main):007:0> opts[:release_apt_repo_url]
=> "http://nightlies.puppet.com/apt"
irb(main):008:0> remote = URI.join( opts[:release_apt_repo_url], deb )
=> #<URI::HTTP http://nightlies.puppet.com/puppet5-nightly-release-xenial.deb>
irb(main):009:0> File.join( opts[:release_apt_repo_url], deb )
=> "http://nightlies.puppet.com/apt/puppet5-nightly-release-xenial.deb"
```

I'll have to update this

Reply:
Fixed in https://github.com/puppetlabs/beaker-puppet/pull/30/commits/036eaa106beeb8d11c095b966dfb9b91983ede48